Makefile

```diff
@@ -0,0 +1,20 @@
+build-dev:
+	cargo build --tests
+
+setup-grcov:
+	cargo install grcov
```

Comment:
Do we also need to install `rustup component add llvm-tools-preview`?

Reply:
Ah whoops! Thanks for the catch!